### PR TITLE
Add venn arxiv

### PR DESCRIPTION
**Pull request by AmberLJC**

### CODE REVIEW

**Comment by AmberLJC** [comment]
It seems I don't have permission to request for a reviewer? Manually request for check @mosharaf

**Comment by mosharaf** [comment]
Weird!

**Review by mosharaf — APPROVED** [review]
lgtm

File: source/_data/SymbioticLab.bib
Comment (mosharaf) [inline source/_data/SymbioticLab.bib:1]: Fix the author field

Comment (mosharaf) [reply source/_data/SymbioticLab.bib:1]: Should be and not comma

**Pull request merged**